### ISSUE TITLE
YOGI Initialization

### ISSUE THREAD

**Issue by PetrochukM** [jettify/pytorch-optimizer#77]
### `exp_avg_sq` Initialization

> "Thus, for YOGI, we propose to initialize the vt based on gradient square evaluated at the initial point averaged over a (reasonably large) mini-batch." 

The initial `exp_avg_sq` should be initialized to the gradient square.

### `exp_avg` Initialization

![image](https://user-images.githubusercontent.com/7424737/76134134-08e12d80-5fd1-11ea-9f0a-4fc0ac8f2e07.png)

The YOGI optimizer `exp_avg` should be initialized to zero instead of `initial_accumulator` based on `m0` above.

**Comment by PetrochukM** [comment]
@avinashsai What do you think? Am I reading the paper wrong?

**Comment by avinashsai** [comment]
@PetrochukM Even I had doubts regarding this. So, I referred to the author's official implementation in  tensorflow (https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/yogi.py).

In line 119, they initialized first and second moments with a constant value.

**Comment by PetrochukM** [comment]
Welp, I wished they'd be more clear. Thank you!

**Comment by jettify** [comment]
Thanks @PetrochukM  @avinashsai ! Should we add comment in code with this note?

**Comment by avinashsai** [comment]
sure, will submit PR with the change
